run-linux.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+# NB. Requires libsteam_api.so on the LD_LIBRARY_PATH
+#     or within ./engine/love/linux/
+
+set -e
+
+GAME_PATH="$(dirname $0)"
```

Comment:
`$(cd "$(dirname "$0")" && pwd)` otherwise you wont get correct path when launching with relative path.

Reply:
I've made it use a different method of getting an absolute path (`realpath`), but thanks for pointing out the issue!